lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -69,6 +69,8 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
   $form['reportback_submissions']['reportback_item'] = array(
     '#weight' => 5,
     '#type' => 'file',
+    '#prefix' => 'Reportback Image',
```

Comment:
@angaither not sure if this is necessarily needed but thought it would be nice to include for the campaign leads for more clarity!